### PR TITLE
Fix unused dependencies using go mod tidy

### PR DESCRIPTION
**Pull request by Pothulapati**
`go mod tidy` updates the mod files.

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,9 +5,8 @@ go 1.12
 require (
 	github.com/Masterminds/goutils v1.1.0 // indirect
 	github.com/Masterminds/semver v1.4.2 // indirect
-	github.com/Masterminds/sprig v2.18.0+incompatible // indirect
```
Comment (grampelberg) [inline go.mod]: Seems weird that sprig would go away?

Comment (Pothulapati) [reply go.mod]: hmmm, but the build works fine?

Comment (simonpasquier) [reply go.mod]: Because it's already present at L19 :)

Comment (Pothulapati) [reply go.mod]: aaah! Right!

**Pull request merged**